src/uni_modules/wot-design-uni/components/wd-drop-menu-item/wd-drop-menu-item.vue

```
@@ -124,12 +189,38 @@ onBeforeUnmount(() => {
+  }
+}
+
 // 模拟单选操作 默认根据 value 选中操作
 function choose(index: number) {
   if (props.disabled) return
   const { valueKey } = props
-  const item = props.options[index]
-  const newValue = item[valueKey] !== undefined ? item[valueKey] : item
+  const item = filterOptions.value[index]
+  const newValue =
+    item[valueKey] !== undefined ? item[valueKey] : item
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)